examples/simple/simple_test.go

```diff
@@ -24,19 +24,38 @@ import (
 	eventstore "github.com/looplab/eventhorizon/eventstore/memory"
 	readrepository "github.com/looplab/eventhorizon/readrepository/memory"
 
+	"testing"
+
 	"github.com/looplab/eventhorizon/examples/domain"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
 )
 
-func Example() {
```

Comment:
The Example name has a purpose, namely as an example in the documentation: https://golang.org/pkg/testing/#hdr-Examples